cfme/base/__init__.py

```diff
@@ -59,6 +61,32 @@ def zone(self):
     def slave_servers(self):
         return self.zone.collections.servers.filter({'slave': True}).all()
 
+    @property
+    def _api_settings_url(self):
+        return '/'.join([self.appliance.rest_api.collections.servers._href, self.sid, 'settings'])
+
+    def get_yaml_config(self):
+        """GET servers/:id/settings api endpoint to query server configuration"""
+        return self.appliance.rest_api.get(url=self._api_settings_url)
+
+    def set_yaml_config(self, data_dict):
```

Comment:
**QUESTION**
Why are `get_yaml_config` and `set_yaml_config` not a property?

```python
@property
def yaml_config(self):
    ...

@yaml_config.setter
def yaml_config(self, data_dict):
    ...
```

Reply:
Mostly because of existing appliance methods not being implemented as properties.  Don't see a reason why they couldn't be, would just require updating all the current uses.  Good suggestion!

@psav thoughts on expanding the scope here a bit to include changing these to properties?

Reply:
I think it's out of the scope. It should be done in another PR anyway.

Reply:
then we agree on all sides! :)